data/families.yml

```diff
@@ -364,7 +364,18 @@ families:
       - "Bonus for 2/3/4/5 Heroes:"
       - "+1%/+3%/+5%/+7% attack."
       - "+1%/+3%/+5%/+7% defense."
-      - "+1%/+3%/+5%/+7% healing." 
+      - "+1%/+3%/+5%/+7% healing."
+  lunaryear:
+    key: lunaryear
+    code: 71
+    description: Lunar Year
+    image: https://i.imgur.com/NWClrk7.png
```

Comment:
Not going to block this PR, but can you add an alpha channel and remove the background from this?